src/main.rs

```diff
@@ -72,7 +76,26 @@ async fn main() {
     dotenv::dotenv().ok();
 
     let framework = StandardFramework::new()
-        .configure(|c| c.prefix("!"))
+        .configure(|c| {
+            c.dynamic_prefix(|ctx, msg| {
+                Box::pin(async move {
+                    let prefixes = get_prefixes();
+
+                    let guild_id = msg.guild(&ctx.cache).await.unwrap().id;
+
+                    if let Some(serde_json::Value::String(prefix)) =
+                        prefixes.get(guild_id.0.to_string())
+                    {
+                        // Command from guild_id.0 with prefix
```

Comment:
These comments are superfluous

Reply:
I thought so, but chose to leave them at the time, will remove.